src/app/drive/types/file-types.ts

```diff
@@ -245,9 +246,9 @@ const excludeUnsupportedVideoExtensions: string[] = [
   'f4b',
 ];
 
-const previewVideoExtensionsGroup: string[] = Object.values(videoExtensions)
-  .flatMap((extensions) => extensions.flat())
-  .filter((extension) => !excludeUnsupportedVideoExtensions.includes(extension));
+const previewVideoExtensionsGroup: string[] = Object.keys(videoExtensions).filter(
+  (extension) => !excludeUnsupportedVideoExtensions.includes(extension),
+);
```

Comment:
Why only use keys instead of all values of videoExtensions?

Reply:
Because I think here it is filtered by keys, so when we try to search for the mp4 extension for example, it tells us that it cannot be displayed because the key is mpeg-4: [https://github.com/internxt/drive-web/blob/aeb2e772457455e50009d308a985ddb0947101b4/src/app/drive/services/file-extension.service.ts](url)